testsuite/forge.py

```diff
@@ -448,14 +448,22 @@ def set_output(self, output: str) -> None:
     def set_debugging_output(self, output: str) -> None:
         self.debugging_output = output
 
-    def format(self) -> str:
-        output = "\n".join([
+    def format(self, context: ForgeContext) -> str:
```

Comment:
Can we print the link before the run as well? This is particularly useful to monitor the test as its running.

Reply:
Yeah will do